core/views.py

```diff
@@ -61,9 +66,12 @@ def get(self, request, *args, **kwargs):
             category__slug=category_slug
             )
 
+        users = User.objects.all()
+
         context = {
             'all_resources': all_resources,
             'category_slug': category_slug,
+            'users': users,
```

Comment:
You do not need to send all the users as context, do not send users in context, it is not safe because you are performing an expensive query to just show the resource creator in template.